spirv_glsl.cpp

```diff
@@ -3565,6 +3565,11 @@ void CompilerGLSL::emit_resources()
 		    (var.storage == StorageClassInput || var.storage == StorageClassOutput) &&
 		    interface_variable_exists_in_entry_point(var.self) && !is_hidden)
 		{
+			if (options.es && get_execution_model() == ExecutionModelVertex && var.storage == StorageClassInput &&
+			    type.array.size() == 1 && to_array_size_literal(type) > 1)
```

Comment:
to_array_size_literal(type) > 1 is probably redundant here. I assume `in vec4 blah[1]` also fails?

Reply:
Yes, you're right, previously I noticed some interface variables were declared as foo[1] in our code base, i'v double checked them, they only appear in output variables. I'll remove this check.